client/prototypes.h

```diff
@@ -24,12 +24,19 @@
 #include <openssl/sha.h>
 #include <openssl/md5.h>
 #include <openssl/evp.h>
-typedef struct metalinkfile{
+
+typedef struct _TDNF_METALINK_URLS_ {
+    char *url;
```

Comment:
url should be const?

Reply:
As discussed, keeping it as it is. Don't want to complicated things.